calico/reference/felix/configuration.md

```diff
@@ -69,7 +69,8 @@ The full list of parameters which can be set is as follows.
 | `RemoveExternalRoutes`            | `FELIX_REMOVEEXTERNALROUTES`            | Whether or not to remove device routes that have not been programmed by Felix. Disabling this will allow external applications to also add device routes. [Default: `true`] | bool |
 | `ReportingIntervalSecs`           | `FELIX_REPORTINGINTERVALSECS`           | Interval at which Felix reports its status into the datastore or `0` to disable. Must be non-zero in OpenStack deployments. [Default: `30`] | int |
 | `ReportingTTLSecs`                | `FELIX_REPORTINGTTLSECS`                | Time-to-live setting for process-wide status reports. [Default: `90`] | int |
-| `RouteTableRanges`                 | `FELIX_ROUTETABLERANGES`                 | Calico programs additional Linux route tables for various purposes. `RouteTableRanges` designates a set of table ranges that Calico is permitted to use. [Default: `1-250,255-10000`] | `<min>-<max>,<min>-<max>,...` |
+| `RouteTableRange`                 | `FELIX_ROUTETABLERANGE`                 | *deprecated in favour of `RouteTableRanges`* Calico programs additional Linux route tables for various purposes.  `RouteTableRange` specifies the indices of the route tables that Calico should use. [Default: `1-250`] | `<min>-<max>` |
```

Comment:
Is "Default 1-250" right now?  I'd expect the default to be unset so that `RouteTableRanges` takes precidence